journal/modernized/01-chapter-1.adoc

```diff
@@ -252,7 +262,8 @@ When I had been with Anna Coulson, from my first going, five years,
 her worthy husband departed this life.
 I was closely with him during his illness,
 and took down the expressions that dropped from him,
-which are printed in one of the volumes of "`Dying Sayings.`"
+which are printed in one of the volumes of [.book-title]#Dying Sayings#^
+footnote:[Piety Promoted in a Collection of the Dying Sayings of Many of the People Called Quakers: With a Brief Account of Some of Their Labors in the Gospel, and Sufferings for the Same.]
```

Comment:
I sort of think the real full book title in the footnote should also be a `[.book-title]#...#`. What do you think?